llvm/lib/Target/RISCV/RISCVInstrInfoZb.td

```diff
@@ -54,6 +54,7 @@ def riscv_unzip   : RVSDNode<"UNZIP",   SDTIntUnaryOp>;
 def riscv_absw    : RVSDNode<"ABSW",    SDTIntUnaryOp>;
 
 // Scalar cryptography
+def clmul         : RVSDNode<"CLMUL",   SDTIntBinOp>;
```

Comment:
This needs to be in `include/llvm/Target/TargetSelectionDAG.td`. Using `RVSDNode` makes it look for `RISCVISD::CLMUL` instead of `ISD::CLMUL`

Reply:
good point. Also, putting it there lets me annotate it as commutative and associative (it is both).